src/NsdPublicationChannelRegistryClient.js

```diff
@@ -1,21 +1,34 @@
 const channelRegistryUri = 'https://api.nsd.no/dbhapitjener/Tabeller/hentJSONTabellData'
 
 const axios = require('axios')
+const RemoteJournalResponse = require('./response/RemoteJournalResponse')
+const RemotePublisherResponse = require('./response/RemotePublisherResponse')
 
-const responseWithBody = (body) => {
+const responseWithBody = (body, type, year) => {
+  const response = []
+  const level = `Nivå ${year}`
+  if (type === 'journal') {
+    body.forEach(item => response.push(new RemoteJournalResponse(item['Tidsskrift id'], year, item['Original tittel'], item.Url, item[level], item.Aktiv, item['Online ISSN'], item['Print ISSN'], item['NPI Fagfelt'], item['Open Access'], item['Språk'], item.Utgiver)))
```

Comment:
Extract methods and use ternary operator

Reply:
Extracted and ternarified method